src/content/certifications/programs/security-automation.json

```diff
@@ -0,0 +1,7 @@
+{
+	"title": "Security Automation",
```

Comment:
💭 Non-blocking thought: perhaps we can add `slug` to the program schema instead of inferring it from the filename. This way it's clear to authors how the page name is derived and they can edit it without renaming the file.

Reply:
Ah great suggestion! 🙌  I was on the fence... I think I lean slightly towards using the filenames as slugs for the following reasons:
- Putting `slug` in content felt duplicative - we have to name the file something, so why not the slug?
- Using the slug as the filename feels like it meshes nicely with NextJS file-based routing under `pages`. It also felt slightly more intuitive in `getStaticPaths` to read the directory contents to get all slugs, without have to read in and parse the file content.
- Some theme styling is based on "program" slugs, so we aren't quite at the point to let authors change the slugs arbitrarily...

All that being said, can see how the relationship between the filename and page URL isn't exactly clear unless you dive into the code. And with that last point on theming being kind of brittle, this is making me think maybe we could have a separate `theme` or something within the content 🤔

Overall my hot take is it'll probably be more clear what the right path is if / when we need to add more certification programs, and for now this at least seems okay. So going to merge this as-is, but very open to changing course here before finalizing the assembly PR 🙇

Reply:
Totally reasonable ☝🏻 I'm sure we can adjust as needed once authors start interacting with the content as well. fwiw I can't imagine these slugs will rarely (if ever) get updated, so not a big concern!